lib/ansible/plugins/callback/default.py

```diff
@@ -113,11 +113,12 @@ def v2_playbook_on_no_hosts_remaining(self):
         self._display.banner("NO MORE HOSTS LEFT")
 
     def v2_playbook_on_task_start(self, task, is_conditional):
-        self._display.banner("TASK [%s]" % task.get_name().strip())
-        if self._display.verbosity > 2:
-            path = task.get_path()
-            if path:
-                self._display.display("task path: %s" % path, color=C.COLOR_DEBUG)
+        if self._display.verbosity > 1 and task.get_name() == 'include':
```

Comment:
Won't this cause no task to ever be printed?  You have nested the 'TASK' banner display under a check that ensures that the task name is 'include' and that the verbosity needs to be greater than 1.